crates/ruff_workspace/src/options.rs

```diff
@@ -2107,6 +2117,9 @@ impl IsortOptions {
             }
         }
 
+        let no_sections = self.no_sections.unwrap_or_default();
+        // TODO: Warn if `no_sections` is true and `sections` or `section-order` is non-empty?
```

Comment:
I think it'd be great to have some of these warnings here prior to merging.

Reply:
I've added a warning for these two situations.